src/python/pants/rules/core/filedeps.py

```diff
@@ -39,31 +39,26 @@ class Filedeps(Goal):
 @goal_rule
```

Comment:
This file only has some light refactoring. We're going to need to change this file once `HydratedTarget.address` is no longer a BFA so that we `await Get[BuildFileAddress](Address)`.